### ISSUE TITLE
Upgrading instances need to be more explicit

### ISSUE THREAD

**Issue by gbiellem** [Particular/ServiceControl#817]
We've have a few cases were user have run the installer but not realizing that after doing so they need to update instances individually via the mgmt util.   

At the moment the PI prompts to run the mgmt util and so does the standalone installer.
**IF** the user does this then they are presented with only one visual queue that an update is required.  i.e. the link that is presented next to the version number

What I'm thinking is 
-  remove the optional launch and make it mandatory.
- Either ask the user do they want to start the update of instances if any are out of date when the mgmt tool launches 
- Or change the link to something more prominent

@sergioc I could use some UX assistance here

**Comment by sergioc** [comment]
> remove the optional launch of the mgmt tool and make it mandatory.

That's fine. The installer conclusion screen just needs to indicate what will happen, something like "The ServiceControl installer will close and open the Management Utility to check for updates".

> Either prompt the the user when the mgmt tool launches -> i.e One or more instances are running on an old version, do you want to run the upgrade? 

Could that get messy if the user only wants to update one of instances in case there's more than 1 set up?

> Or change the link to something more prominent

This is probably the better option, since we then leave it up to the user to decide what to upgrade.

I don't think it's worth yet to explore other notification means without trying what looks like quick wins above.

**Comment by gbiellem** [comment]
@sergioc  so how do you think we should go about making the link more prominent?

**Comment by gbiellem** [comment]
Thanks @sergioc.   I prefer No 2.   

@johnsimons what do you think? Is just flipping the link to a button enough?

**Comment by johnsimons** [comment]
I am 👍 to make that link more obvious, but, just to be sure, we also doing the first two items right ?

**Comment by johnsimons** [comment]
I have updated the description to convey that everything should be done

**Comment by gbiellem** [comment]
I've added a 4th think "Update PI and Installer screens to be more explict about what needs to happen next"

And with number 2 I think we could do it but maybe change the text to " One or more instances are running on an old version."  and just make it an OK button to dismiss rather than launching into the upgrade.  This would solve @sergioc's point about what do when there are 2 or more instances   I also think if that popup was in place i'd want a "Don't propmpt me again" option on it

**Comment by johnsimons** [comment]
> I also think if that popup was in place i'd want a "Don't propmpt me again" option on it

You mean just for that specific version ?

**Comment by mikeminutillo** [comment]
Could 2 just be a banner at the top rather than a pop-up? That way you don't need to have a dismissal button.

**Comment by johnsimons** [comment]
> Could 2 just be a banner at the top rather than a pop-up?

I would go for that as long as the first time we make it a popup so they need to actively dismiss it.

**Comment by gbiellem** [comment]
> You mean just for that specific version ? 

Yes.  but maybe that could just be the default behavior if we went with a banner as well.
Regarding placement, I assume it would sit just under the "Deployed Instances" heading

**Comment by johnsimons** [comment]
> I assume it would sit just under the "Deployed Instances" heading

Yep, outside the scrolling area

**Comment by sergioc** [comment]
> I have updated the description to convey that everything should be done

@johnsimons hang on. I suggested making the upgrade link more obvious. That alone will already make the upgrade notification much more visible compared to what we have now. There's no need to add more noise until we confirm users still have the problem.

**Comment by gbiellem** [comment]
Firstly I like the change from the link to a button.  I think is looks better and is definitely more prominent

However the issue is that we are have people not picking up the fact that they need to do anything in that screen and just making the button more prominent isn't guiding the user to complete the upgrade.
I'm not saying the banner or popup or combo of the two is the correct approach but I don't believe just the link to button change will resolve the issue

**Comment by johnsimons** [comment]
Also the user has to see the upgrade button, and since we have a list of endpoints, the upgrade button may not be in view, so this is taken care by the banner in the non scroll-able area.

The other 2 options are also required:
- Automatically launching the management utility -> otherwise they will not run it;
- Popup dialog first time if there are out of date instances -> this one is to complement the previous task, it ensures a fluid continuation of installing the product;
